README.md

```diff
@@ -54,6 +54,34 @@ Here
 * `--url` specifies a payload URL;
 * `--output-file` specifies a file to write the scan report to.
 
+### Scan a Local Unpacked Image Bundle
+
+The `scan local` subcommand allows you to scan a local unpacked image bundle for FIPS compliance. This is particularly useful for testing or analyzing local images that have been unpacked using tools like `umoci`.
+
+#### Usage
+
+To scan a local unpacked image bundle, use the following command:
+
+```sh
+./check-payload scan local --path /path/to/local/bundle
```

Comment:
Will `scan local --path` scan unpacked operator bundles (which include the `/manifests` and `/metadata` directories), and/or unpacked "operand" images, which are "normal" images with the usual expected directory structure (`/etc/`, `/usr/`, `/sbin` and so forth).

Apologies if I've misunderstood something. I am new to this tool. :)

Reply:
`check-payload scan local` should scan any unpacked file system, it will walk the filesystem from the provided "--path" and should work as it would after the (skipped) `podman mount` phase in the `scan operator` work flow.